python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/scenarios/auto_materialize_policy_scenarios.py

```diff
@@ -297,7 +297,9 @@ def with_auto_materialize_policy(
             assets=lazy_assets_nothing_dep,
             unevaluated_runs=[run(["asset1"])],
             expected_run_requests=[run_request(asset_keys=["asset2", "asset3"])],
+            asset_selection=AssetSelection.keys("asset2", "asset3"),
```

Comment:
Bugfix: Need to add asset selection since asset1 has no AMP, so `do_sensor_scenario` doesn't target asset1